src/legacy/ui/public/_index.scss

```diff
@@ -19,7 +19,6 @@
 @import './exit_full_screen/index';
 @import './field_editor/index';
 @import './kbn_top_nav/index';
-@import './markdown/index';
```

Comment:
By removing the import from here, how is it being compiled?

Reply:
I added it to `src/legacy/core_plugins/kibana_react/public/index.scss`

Reply:
Ahh, see it now, thx

Reply:
done